src/applications/vaos/project-cheetah/redux/actions.js

```diff
@@ -449,6 +455,41 @@ export function onCalendarChange(selectedDates, pageKey) {
   };
 }
 
+export function openContactFacilitiesPage() {
```

Comment:
This is similar to the `openFacilitiesPage` action creator, except that there's no filtering by type of care. We just want to get any facilities where a user is registered.